.github/actions/gradle/experiment-1/action.yml

```diff
@@ -103,8 +103,8 @@ runs:
         
         echo "buildScanFirstBuild=$BUILD_SCAN_1" >> $GITHUB_OUTPUT
         echo "buildScanSecondBuild=$BUILD_SCAN_2" >> $GITHUB_OUTPUT
```

Comment:
I would remove this two statements, now that we have the rich summary:
<img width="478" alt="image" src="https://user-images.githubusercontent.com/231070/204192686-82c34032-b61f-4cc3-8b6c-2ab32a5c24bd.png">

Reply:
Hm, I catually do not see it in the output anymore, but I wonder why not, given the statements are still in the yml file.

Reply:
Those are actually not printed but actual outputs of the composite step.
I would still keep them as it might be useful for automation imo (ie. subsequent calls to the API).

Please let me know your preference.

Reply:
Ah. I see. Let's leave it then as it is on this PR.